devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy

```diff
@@ -50,6 +50,7 @@ metadata {
 
 		// Commercial Electric
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "ETI", model: "Zigbee CCT Downlight", deviceJoinName: "Commercial Electric Can Tunable White"
+		fingerprint manufacturer: "ETI", model: "Zigbee CCT Downlight", deviceJoinName: "Commercial Electric LED Smart Recessed Zigbee Downlight" // raw description: 01 0104 0102 00 07 0000 0003 0004 0005 0006 0008 0300 01 0019
```

Comment:
@KKlimczukS I see now that this device has the same manufacturer/model as the existing fingerprint. Thus you really shouldn't need to do anything to the DTH and it should join with the name `Commercial Electric Can Tunable White`. Alternatively you can change the join name.